src/lib/swapper/types.ts

```diff
@@ -151,6 +151,22 @@ export enum SwapErrorType {
 
 export type FromOrXpub = { from: string; xpub?: never } | { from?: never; xpub: string }
 
+export type CowSwapOrder = {
```

Comment:
Would prefer this still located in a CoW specific domain but meh, nitpick as far as this PR goes